### DISCUSSION TITLE
Grid events and persisting grid order

### DISCUSSION THREAD

**Discussion by slinky-bass** [cuire/svelte-grid-extended#72] [Q&A] [answered]
Apologies in advance if this is an obvious question, I'm still pretty new to Svelte.

I see that the original repo `svelte-grid` supported serialization of the layout items order/positions for persisting the users selection in local storage. Is there a way to get the same sort of info, i.e.  the updated item layout order, to do the same in `svelte-grid-extended`?

**Comment by cuire** [comment, marked as answer]
Hey, here a quick demo how you can achive that, hope it helps
https://svelte.dev/repl/effd88614c0c4372864ebeb582415a21?version=4.1.1

**Reply by slinky-bass** [reply]
Thank you so much for the quick reply and repl. I got it working, but your example using the store is much cleaner.
